src/DSharp.Compiler/ScriptModel/Symbols/MethodSymbol.cs

```diff
@@ -84,15 +89,15 @@ public SymbolImplementation Implementation
             }
         }
 
-        public bool IsAliased => string.IsNullOrEmpty(TransformName) == false;
+        public bool IsAliased => !string.IsNullOrEmpty(TransformName);
 
         public bool IsExtension
```

Comment:
This will die in the CSharp 3 work, because it's a huge hack.

Reply:
It will be extremely difficult but yes, I'm up for doing that